client/landing/jetpack-cloud/components/activity-card/index.jsx

```diff
@@ -172,9 +179,25 @@ class ActivityCard extends Component {
 					<Button
 						href={ backupRestorePath( siteSlug, activity.rewindId ) }
 						className="activity-card__restore-button"
+						disabled={ doesRewindNeedCredentials }
```

Comment:
we should add `event.preventDefault` to stop the action if the user click on the button when it's disabled.

```
onClick={ ( event ) => {
					( disabledRestore || doesRewindNeedCredentials ) && event.preventDefault();
```